frontend/webpack.config.js

```diff
@@ -79,8 +81,20 @@ module.exports = {
     }]
   },
   resolve: {
+    plugins: [
+      PnpWebpackPlugin,
+    ],
+    alias: {
+      // Make ~tracim_frontend_lib work in stylus files
+      '~tracim_frontend_lib': path.dirname(path.dirname(require.resolve('tracim_frontend_lib')))
```

Comment:
I reused the notation used in the doc of stylus-native-loader: https://www.npmjs.com/package/stylus-native-loader

Look for the following in this page:

>  ```javascript
>  resolve: {
>    // All aliases are used for Stylus @import and @require path resolution
>    alias: {
>      // Maps @import('~styl/*') to @import('/path/to/src/styl/*')
>      '~styl': path.join(__dirname, 'src/styl'),
>  
>      // Maps @import('mixins') to @import('/path/to/src/styl/mixins')
>      'mixins$': path.join(__dirname, 'src/styl/mixins'),
>    },
>  },
>  ```

See also the alias declaration section that defines it properly in tracim's own webpack configs. 

This seems a standard notation to denote an import from a node module in stylus, this will help a new dev looking at the stylus file/

We have no choice: the old way hard codes the location of tracim_frontend_lib, but with yarn the location is not in node_modules anymore. The old way does not work anymore and maybe worked by luck (it was dependent on NPM's specific behavior). The new way works with regular NPM though :-)

Reply:
Ok for the 2 path.dirname(). Since it's a bit confusing, I'd advise a comment to explain it

About the "~", if it's in the module documentation, I'm fine with it

Reply:
you can resolve that conversation